public/scripts/editor/js/fc/load-project-files.js

```diff
@@ -22,56 +22,26 @@ define(["jquery", "constants", "text!fc/stay-calm/index.html", "text!fc/stay-cal
     var projectFilesUrl = "//" + window.location.host + "/initializeProject";
     var request;
     var projectTitle = project && project.title;
+    var root = project && project.root;
     var defaultProject = false;
-    var defaultPath;
     var files;
 
-    if(!projectTitle) {
+    if(!projectTitle || !root) {
```

Comment:
realistically, this condition should never eval to true because that would be very bad, I wonder if there is even a point checking for this